requirements_caiman.txt

```diff
@@ -0,0 +1,30 @@
+bokeh
+coverage
+Cython
+future
+h5py
+holoviews
+ipykernel
+ipython
+ipyparallel
+jupyter
+keras
+matplotlib
+mypy
+numpy==1.21.0
+numpydoc
+opencv-python
+peakutils
+pims
+psutil
+pynwb
+pyqtgraph
+scikit-image
+scikit-learn
+scipy
+# tensorflow
+tifffile
+tk
+tqdm
+yapf
+caiman @ git+https://github.com/datajoint-company/caiman
```

Comment:
@CBroz1 Is this the strategy that we will be employing to install CaImAn going forward?

Reply:
Yes, I think this is a good middle-ground to permit pip installation even when supported packages haven't done the same. We may be behind, however, in documenting this method for new users

Reply:
I'm back to caiman installation. I'll resolve this installation issue ASAP.

Reply:
Let's remove this file since it's not relevant to the PR.

Reply:
Sure - I've removed this requirements file